packages/calcite-components/src/components/button/button.e2e.ts

```diff
@@ -572,16 +558,11 @@ describe("calcite-button", () => {
         </style>
         <div>${buttonSnippet}</div>`,
       });
-      buttonEl = await page.find("calcite-button >>> button");
-      await buttonEl.focus();
-      await page.waitForChanges();
-      buttonFocusStyle = await buttonEl.getComputedStyle();
-      expect(buttonFocusStyle.getPropertyValue("background-color")).toEqual(overrideStyle);
-
+      const buttonEl = await page.find("calcite-button >>> button");
       await buttonEl.hover();
       await page.waitForChanges();
-      buttonHoverStyle = await buttonEl.getComputedStyle();
-      expect(buttonHoverStyle.getPropertyValue("background-color")).toEqual(overrideStyle);
+      const buttonFocusStyle = await buttonEl.getComputedStyle();
+      expect(buttonFocusStyle.getPropertyValue("background-color")).toEqual(overrideStyle);
     });
   });
```

Comment:
Per the [guidelines](https://github.com/Esri/calcite-design-system/wiki/tokens-component#e2e), can you add `themed` tests?